sys/mips/mediatek/mtk_machdep.c

```diff
@@ -258,12 +258,15 @@ platform_start(__register_t a0 __unused, __register_t a1 __unused,
 
 	printf("FDT DTB  at: 0x%08x\n", (uint32_t)dtbp);
 
+	printf("CPU   model: %s\n", mtk_soc_get_cpu_model());
```

Comment:
I think that cpu_model could be used here. It's already populated at this point.